specification/maps/data-plane/Geolocation/preview/1.0/geolocation.json

```diff
@@ -56,7 +56,7 @@
     "/geolocation/ip/{format}": {
       "get": {
         "description": "\n**Applies to**: S0 and S1 pricing tiers.\n\n\nThis service will return the ISO country code for the provided IP address. Developers can use this information  to block or alter certain content based on geographical locations where the application is being viewed from. \n\n\n__Note:__ This service returns results from IANA and does not necessarily reflect the views of Microsoft  Corporation.",
-        "operationId": "Geolocation_GetIPToLocationPreview",
+        "operationId": "GetIPToLocationPreview",
```

Comment:
Remove Preview from this name

Reply:
I renamed this in a later commit. It's now GetLocation.